ci/ci/github.py

```diff
@@ -132,6 +132,30 @@ def __init__(self, exception, attributes=None):
 DO_NOT_MERGE = {STACKED_PR, WIP}
 
 
+def clone_or_fetch_script(repo):
+    return """
+{ RETRY_FUNCTION_SCRIPT }
+
+function clone() {{
+    dir=$(mktemp -d)
+    git clone {shq(repo)} $dir
+    for x in $(ls -A $dir); do
+        if [ -e "$dir/$x" ]; then mv -- "$dir/$x" ./; fi
```

Comment:
Why do you need to check the existence of `$dir/$x`? Shouldn't it exist since you list the directory?

Reply:
I copied this from stack overflow and honestly am not sure. I'll remove it.